homeassistant/components/rehlko/strings.json

```diff
@@ -31,6 +31,17 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "connected": {
+        "name": "Connected"
+      },
```

Comment:
Let's use the device class translation for this one

Reply:
Thanks, that's a nice way to do this.